packages/math-input/src/components/keypad/button.tsx

```diff
@@ -44,60 +49,76 @@ const styles = StyleSheet.create({
         boxSizing: "border-box",
         borderRadius: 7,
         border: "2px solid transparent",
-    },
-    outerBoxHover: {
-        border: `2px solid ${Color.blue}`,
-    },
-    outerBoxPressed: {
-        border: "2px solid #1B50B3",
-    },
-    clickable: {
-        width: "100%",
-        height: "100%",
-        boxSizing: "border-box",
+        ":hover": {
+            border: `2px solid ${Color.blue}`,
+        },
+        ":active": {
+            border: "2px solid #1B50B3",
+        },
     },
 });
 
-export type Props = {
+type OwnProps = {
     onPress: () => void;
     children: React.ReactNode;
     style?: StyleType;
     tintColor?: string;
+    keyConfig: KeyConfig;
+};
+
+type ReduxProps = {
+    gestureManager: GestureManager;
 };
 
-export default class Button extends React.Component<Props> {
+export type Props = OwnProps & ReduxProps;
+
+class Button extends React.Component<Props> {
```

Comment:
FWIW, given this only has a `render` function, it could itself before a simple pure functional component.